lib/mongo/database.rb

```diff
@@ -155,11 +158,11 @@ def collections
     def command(operation, opts = {})
       preference = ServerSelector.get(opts[:read] || ServerSelector::PRIMARY)
       server = preference.select_server(cluster)
-      Operation::Commands::Command.new({
+      Operation::Commands::Command.new(
```

Comment:
I'm also not sure if commands should be executed in the context of a session or not